modules/cassandra/store/src/main/java/org/apache/ignite/cache/store/cassandra/session/CassandraSessionImpl.java

```diff
@@ -50,6 +52,74 @@
  * Implementation for {@link org.apache.ignite.cache.store.cassandra.session.CassandraSession}.
  */
 public class CassandraSessionImpl implements CassandraSession {
+    /**
+     * Simple container for Cassandra session and its generation number.
+     */
+    private static class WrappedSession {
```

Comment:
Considering that you have moved WrapperPrepareStatement in a seperate file, would you want to do the same for WrappedSession?

Reply:
Yeah, actually it'll make code more clean